### ISSUE TITLE
[Bug/Glitch] no overworld music or menu music

### ISSUE THREAD

**Issue by agariLMAO** [Wilyicaro/Legacy-Minecraft#502]
### Legacy4J Version

1.20.1-1.7

### Mod Loader Version

Fabric Loader 0.15.11

### Bug/Glitch Occurrence Context

i was messing around to see if i could make a version of a modpack in 1.20.1 and i noticed there was no music.

### What happened wrong?

so far only music has played when going underwater.  i disable all mods except for legacy4j and theres still no music. it could be broken afaik.

**Comment by Wilyicaro** [comment]
No, this isn't broken, I literally played only 1 minute in a world and this played

**Comment by agariLMAO** [comment]
> No, this isn't broken, me after literally only 1 minute in a world this played

nah it still happens, except i found out it happens when programmer art is applied

**Comment by agariLMAO** [comment]
like the music is off whenever programmer art is enabled

**Comment by Nobody48213** [comment]
> like the music is off whenever programmer art is enabled

Turn off programmer art

**Comment by agariLMAO** [comment]
> > like the music is off whenever programmer art is enabled
> 
> Turn off programmer art
well i'd rather had programmer art on, plus its kinda required for a modpack im working on

**Comment by JuniperIsSleepy** [comment]
this literally makes no sense, programmer art does not affect music

**Comment by agariLMAO** [comment]
after that there was no music
expect when you go underwater

**Comment by agariLMAO** [comment]
all i know for a fact that theres mostly no overworld music and no menu music at all when programmer art is enabled

**Comment by agariLMAO** [comment]
https://discord.com/channels/1203542640008892498/1203542640470138908/1261349528666378421
